tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -400,6 +400,103 @@ TILEDB_EXPORT int32_t tiledb_query_get_status_details(
     tiledb_query_t* query,
     tiledb_query_status_details_t* status_details) TILEDB_NOEXCEPT;
 
+/* ****************************** */
+/*          QUERY CONDITION       */
+/* ****************************** */
+
+/**
+ * Allocates a TileDB query condition value object.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * uint32_t value = 5;
+ * tiledb_query_condition_t* query_condition;
+ * tiledb_query_condition_alloc_value(
+ *    ctx,
+ *    "longitude",
+ *    &value,
+ *    sizeof(value),
+ *    TILEDB_LT,
+ *    &query_condition);
+ *
+ * @endcode
+ *
+ * @param ctx The TileDB context.
+ * @param attribute_name The attribute name.
+ * @param condition_value The value to compare against an attribute value.
+ * @param condition_value_size The byte size of `condition_value`.
+ * @param op The comparison operator.
+ * @param query_condition The query condition to create
+ * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
+ */
+TILEDB_EXPORT int32_t tiledb_query_condition_alloc_value(
```

Comment:
Do I understand correctly that this does the same thing as the sequence of `tiledb_query_condition_alloc` then `tiledb_query_condition_init`? If so I'd like a different name, one that doesn't use `alloc` but rather `create`.

Maybe `tiledb_query_condition_create_field_op_value`.

Reply:
Yep. I covered this in a longer comment.